sdk/ml/azure-ai-ml/azure/ai/ml/requirements.txt

```diff
@@ -0,0 +1,26 @@
+# NOTE: To avoid breaking changes in a major version bump, all dependencies should pin an upper bound if possible.
```

Comment:
This file shouldn't be in the package - please make sure all of the dependencies are correctly updated in setup.py

Reply:
It's gone.